src/components/TodoItem.jsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react'
+import { AiOutlineClose } from 'react-icons/ai';
+
+function TodoItem({title, done, id, deleteTodo, toggleTodoStatus, updateTodo}) {
```

Comment:
It's better to use the arrow function for this component as you have decided to use it on the other functions inside it! and for every component like this one you should apply the same practice.